Assets/MRTK/SDK/Features/UX/Scripts/StateVisualizer/StateAnimatableProperties/IStateAnimatableProperty.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI.Interaction
```

Comment:
Make sure that all experimental classes are in a separate experimental namespace, following the [guide here](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/ExperimentalFeatures.html#experimental-code-should-be-in-a-special-namespace)